src/components/HelpScoutBeacon/HelpScoutBeacon.js

```diff
@@ -337,25 +335,91 @@ ToggleDialogueButton.propTypes = {
 
 const OptInDialogue = React.memo(({ onClose, onOptIn, optedIn, ...styles }) => {
   const theme = useTheme()
-  const { below } = useViewport()
+  const { below, above } = useViewport()
 
   return (
     <animated.div {...styles}>
-      <Wrapper theme={theme}>
-        <Header theme={theme}>
+      <aside
+        css={`
+          background: ${theme.helpSurface};
+          position: absolute;
+          bottom: ${-2 * GU}px;
+          right: ${-2 * GU}px;
+          width: 100vw;
+          height: 100vh;
+          z-index: 1;
+          overflow: hidden;
+          display: flex;
+          flex-direction: column;
+
+          ${above('medium') &&
+            `
+                bottom: calc(-40px - ${4 * GU}px);
+                width: 336px;
+                height: 482px;
+                position: unset;
+                box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.15);
+                border-radius: ${RADIUS}px;
```

Comment:
```suggestion
              bottom: calc(-${5 * GU}px - ${4 * GU}px);
              width: ${42 * GU}px;
              height: ${60.25 * GU}px;
              position: unset;
              box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.15);
              border-radius: ${RADIUS}px;
```

I think we could try `60` rather than `60.25`?